natives.json

```diff
@@ -62403,15 +62403,15 @@
 			"build": "1207"
 		},
 		"0x849BD6C6314793D0": {
-			"name": "_0x849BD6C6314793D0",
+			"name": "_GET_PED_VEHICLE_CARRIAGE",
```

Comment:
By looking at the disassembly and the scripts, I feel like this is more of a "_GET_TRANSPORT_PED_IS_SEATED_ON" native (seeing that there is ``TransportQueryHelpers::GetTransportPedIsSeatedOn``).

Reply:
Yes, `_GET_TRANSPORT_PED_IS_SEATED_ON` is better than `_GET_PED_VEHICLE_CARRIAGE`.